### ISSUE TITLE
Questions about the FGVC experiments.

### ISSUE THREAD

**Issue by schuy1er** [grypesc/AdaGauss#6]
Hi,
Thanks for your excellent work, which has been very inspiring. However, I encountered some issues when reproducing the aircraft 20 task experiment based on the pre-trained model, and the accuracy is somewhat low. Are the hyperparameters in the script you provided accurate? For example, are the rotation and kd parameters correct? Looking forward to your reply!

**Comment by grypesc** [comment]
Hey, you are right, I uploaded wrong script for Aircrafts T=20.  I looked into my old logs and here are correct arguments and corresponding result:

[args-2024-04-30-15-25-23.txt](https://github.com/user-attachments/files/21559119/args-2024-04-30-15-25-23.txt)

[stdout-2024-04-30-15-25-23.txt](https://github.com/user-attachments/files/21559120/stdout-2024-04-30-15-25-23.txt)

Let me know if that works!

**Comment by schuy1er** [comment]
Thanks a lot! I will update after trying it.
